templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -1293,6 +1329,7 @@ Outputs:
   DBMaster:
     Description: The RDS ARN to use when creating a Data Center standby stack
     Value: !If [DBEngineAurora, '', !If [NotStandbyMode, !Sub ["arn:${AWS::Partition}:rds:${AWS::Region}:${AWS::AccountId}:db:${DB}", {DB: !Ref "DB"}], !Ref "AWS::NoValue"]]
+    Value: !If [DBEngineAurora, '', !If [NotStandbyMode, !Sub ["arn:aws:rds:${AWS::Region}:${AWS::AccountId}:db:${DB}", {DB: !Ref "DB"}], !Ref "AWS::NoValue"]]
```

Comment:
This looks like it's been duplicated too

Reply:
Fixed duplicate lines